### PR TITLE
Fixed some logout crashes

### PR DESCRIPTION
**Pull request by LogicParrot**

### CODE REVIEW

**Comment by SphinxC0re** [comment]
Looks good to me

File: src/ClientHandle.cpp [hunk 1]
```diff
@@ -174,6 +173,8 @@ void cClientHandle::Destroy(void)
 		cWorld * World = m_Player->GetWorld();
 		if (World != nullptr)
 		{
+			m_Player->StopEveryoneFromTargetingMe();
```
Comment (LogicParrot) [inline src/ClientHandle.cpp]: The key change is this line.

Comment (SphinxC0re) [inline src/ClientHandle.cpp]: Because when a player is logged out, atm mobs are still targeting him?

Comment (tigerw) [inline src/ClientHandle.cpp]: #weakptrmasterrace

Comment (LogicParrot) [inline src/ClientHandle.cpp]: @SphinxC0re yes.

**Pull request merged**